safe_app/src/tests/mod.rs

```diff
@@ -97,7 +196,7 @@ fn get_access_info() {
 
     unwrap!(run(unsafe { &*app }, move |client, context| {
         context.get_access_info(client).then(move |res| {
-            let info = unwrap!(res);
+            let info = unwrap!(res).1;
             assert!(info.contains_key(&"_videos".to_string()));
```

Comment:
we should no longer be returning this no? It should all be in token

Reply:
And what we want to do is check we get the latest token, and that it is valid for latest settings

Reply:
>we should no longer be returning this no? It should all be in token

These are the old containers. We should be removing them when we bring in Labels.